my-app/src/Components/Account.tsx

```diff
@@ -2,40 +2,25 @@ import React, { useState } from "react";
 import ToolTip from "./ToolTip";
 import AccountHover from "./AccountHover";
 import { useNavigate } from "react-router-dom";
+import { AccountProp } from "../TypesDefines/types";
 
-type Account = {
-  sidepanelOperation: (x: boolean) => void;
-  sidePanel: Boolean;
-};
-
-function Account({ sidepanelOperation, sidePanel }: Account) {
+function Account({ sidepanelMinimize, sidePanel }: AccountProp) {
```

Comment:
use `onSidePanelMinimize` instead. Can we have a better name for this?

Reply:
are we using `sidePanel` prop?

Reply:
sidePanel is using as prop